execution/union_all.go

```diff
@@ -64,9 +63,12 @@ func (node *UnifiedStream) Next(ctx context.Context) (*Record, error) {
 
 	changeSubscriptions := make([]*storage.Subscription, len(node.sources))
 	for i := range node.sources { // TODO: Think about randomizing order.
-		key := octosql.MakeString(fmt.Sprint(i))
+		// Here we try to get a record from the i'th source stream.
```

Comment:
If you decide to change "i" to "sourceIndex" then don't forget to include the change in the comment please :D